app/routes/groups.tsx

```diff
@@ -1,26 +1,45 @@
-import { Card } from '~/components/ui/containers';
 import { useGroups } from '~/hooks/useGroups';
 import { Link } from '@remix-run/react';
-import { LinkButton } from '~/components/ui/forms';
+import { Button } from '~/components/ui/button';
+import { Image } from '~/components/ui/images';
 
 export default function Group() {
   const groups = useGroups();
   return (
-    <>
-      <Card>
-        <ul className="max-w-md space-y-1 text-gray-500 list-disc list-inside dark:text-gray-400">
+    <div className="bg-primary">
+      <div className="m-auto max-w-screen-xl">
+        <ul className="w-full flex flex-wrap justify-center text-secondary">
           {groups?.map((group) => {
             return (
-              <li key={group.id}>
-                <Link to={group.id}>{group.name}</Link>
+              <li className="flex flex-wrap justify-center my-2 mx-auto md:m-2" key={group.id}>
+                <Link to={group.id}>
+                  <div className="border-slate-100 border-2 rounded-2xl  m-1 px-4 py-4 flex flex-col justify-center items-center md:w-80 hover:transform hover:scale-105 transition-transform duration-300">
+                    <div className="w-[13.13rem] h-[13.13rem]">
```

Comment:
NIT: Can we use w-48 h-48 instead of the custom ones?

Reply:
Updated to w-52 h-52 for 13rem